plugins/napari_ADS/_widget.py

```diff
@@ -402,6 +564,95 @@ def _on_load_mask_button_click(self):
             "associated_myelin_mask_name"
         ] = myelin_mask_name
 
+    def _on_remove_axons_click(self):
+        """Handles the click event of the 'Toggle axon removal' button.
+
+        Switches the state of the remove_axon_state attribute, which is used to determine whether the user can click axons to remove
+
+        Returns:
+            None
+        """
+
+        if not self.image_loaded_after_plugin_start:
+            self.show_info_message("Please load an image first. If you loaded an image and are seeing this, you loaded the image prior to the plugin. Please remove and relopen the image and masks.")

```

Comment:
Do you have to restrict to images loaded after the plugin has been loaded? Maybe we can add a check when the `Axon Removal` or `show axon morphometrics` features are toggled and add the callback at that moment

Reply:
>Do you have to restrict to images loaded after the plugin has been loaded?

In my initial implementation, this is because our custom method `_on_layer_added` activates `layer.mouse_drag_callbacks.append(self._on_image_click)`, but now that I think of it, you're right that we can maybe add it to the buttons I added. I'll test it out!

Reply:
I think I prefer to keep it this way for now - I like a single handler for when images are loaded, and we can setup / do checks there for all of our functions later. Maybe later when we tackle #871 we could have this handler initiated when the plugin is loaded (i.e. go through available layers that were loaded prior to the plugin and call this handler for each of them), but I'll have to write more extensive tests for that.